cmd/main.go

```diff
@@ -3,52 +3,70 @@ package main
 import (
 	"context"
 	"log"
-	"log/slog"
+	"os"
 
 	"github.com/alexboor/lbx-telebot/internal"
 	"github.com/alexboor/lbx-telebot/internal/cfg"
 	"github.com/alexboor/lbx-telebot/internal/handler"
 	"github.com/alexboor/lbx-telebot/internal/model"
 	"github.com/alexboor/lbx-telebot/internal/storage/postgres"
+	"github.com/joho/godotenv"
 	tele "gopkg.in/telebot.v3"
+	"log/slog"
 )
 
 func main() {
+	if err := godotenv.Load(); err != nil {
+		log.Fatalf("Error loading .env file: %s", err)
+	}
+
 	cfg.InitLogger()
 
+	opts := &slog.HandlerOptions{AddSource: true, Level: slog.LevelDebug}
+	logger := slog.New(slog.NewJSONHandler(os.Stdout, opts))
+	slog.SetDefault(logger)
+
 	slog.Info("starting...")
 	defer slog.Info("finished")
 
 	config := cfg.New()
 	ctx := context.Background()
 
+	slog.Info("Initializing PostgreSQL connection")
 	pg, err := postgres.New(ctx, config.Dsn)
 	if err != nil {
-		log.Fatalf("error connection to db: %s\n", err)
+		log.Fatalf("error connection to db: %s", err)
 	}
+	slog.Info("Connected to PostgreSQL")
 
 	h := handler.New(pg, config)
 	if err != nil {
-		log.Fatalf("error create handler: %s\n", err)
+		log.Fatalf("error create handler: %s", err)
 	}
 
-	opts := tele.Settings{
+	optsTele := tele.Settings{
 		Token:  config.Token,
 		Poller: &tele.LongPoller{Timeout: internal.Timeout},
 	}
 
-	bot, err := tele.NewBot(opts)
+	bot, err := tele.NewBot(optsTele)
 	if err != nil {
-		log.Fatalf("error create bot instance: %s\n", err)
+		log.Fatalf("error create bot instance: %s", err)
 	}
+	slog.Info("Bot instance created")
 
 	// getting information about profiles
 	uniqUserIds := map[int64]struct{}{}
 	for _, chatId := range config.AllowedChats {
+		// Convert old group ID to supergroup ID if needed
+		if chatId > 0 {
+			chatId = -1000000000000 + chatId
```

Comment:
what is the purpose?
add comment about this magic number and move it to constant